src/react/backend/location/PauseAndResume.tsx

```diff
@@ -0,0 +1,178 @@
+import { Icon, Loader } from '@scality/core-ui';
+import { Box } from '@scality/core-ui/dist/next';
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppState } from '../../../types/state';
+import { notFalsyTypeGuard } from '../../../types/typeGuards';
+import { handleClientError, networkEnd, networkStart } from '../../actions';
+import { useManagementClient } from '../../ManagementProvider';
+import { InlineButton } from '../../ui-elements/Table';
+import type { InstanceStatus } from '../../../types/stats';
+
+export const PauseAndResume = ({ locationName }: { locationName: string }) => {
+  const dispatch = useDispatch();
+  const instanceId = notFalsyTypeGuard(
+    useSelector((state: AppState) => state.instances.selectedId),
+  );
+  const managementClient = useManagementClient();
+  const queryClient = useQueryClient();
+  const invalidateInstanceStatusQueryCache = async () => {
+    await queryClient.invalidateQueries(['instanceStatus', instanceId]);
+    await queryClient.refetchQueries(['instanceStatus', instanceId]);
+    dispatch(networkEnd());
+  };
+
+  const zenkoClient = useSelector((state: AppState) => state.zenko.zenkoClient);
+
+  const pauseReplicationSiteMutation = useMutation(
+    (locationName: string) => {
+      dispatch(networkStart('Pausing Async Metadata updates'));
+      return zenkoClient.pauseCrrSite(locationName);
+    },
+    {
+      onSuccess: () => {
+        invalidateInstanceStatusQueryCache();
+      },
+      onError: () => {
+        dispatch(networkEnd());
+      },
+    },
+  );
+
+  const resumeReplicationSiteMutation = useMutation(
+    (locationName: string) => {
+      dispatch(networkStart('Resuming Replication workflow'));
+      return zenkoClient.resumeCrrSite(locationName);
+    },
+    {
+      onSuccess: () => {
+        invalidateInstanceStatusQueryCache();
+      },
+      onError: () => {
+        dispatch(networkEnd());
+      },
+    },
+  );
+
+  const pauseIngestionSiteMutation = useMutation(
+    (locationName: string) => zenkoClient.pauseIngestionSite(locationName),
+    {
+      onSuccess: () => {
+        invalidateInstanceStatusQueryCache();
+      },
+    },
+  );
+
+  const resumeIngestionSiteMutation = useMutation(
+    (locationName: string) => zenkoClient.resumeIngestionSite(locationName),
+    {
+      onSuccess: () => {
+        invalidateInstanceStatusQueryCache();
+      },
+    },
+  );
+
+  const {
+    data: instanceStatus,
+    status,
+    isFetching: loading,
+  } = useQuery({
+    queryKey: ['instanceStatus', instanceId],
+    queryFn: () => {
+      return managementClient?.getLatestInstanceStatus(
+        instanceId,
+      ) as InstanceStatus;
+    },
+    onError: (error) => {
+      dispatch(handleClientError(error));
+    },
+    refetchOnMount: false,
+    refetchOnWindowFocus: false,
+  });
```

Comment:
I would extract this query into a queries object similarly as we do for other features. That will enable simpler reuse of the queries later on